template.env

```diff
@@ -22,3 +22,5 @@ SMARTHOST_ALIASES=
 
 # comma separated list of organizations to create (e.g ORGANIZATIONS="ORG1, ORG2, ORG3")
 ORGANIZATIONS=
+# Host folder containing the MISP feeds generated by misp-tools
+FEEDS_MOUNT_POINT=
```

Comment:
Can we make this functionality optional? Aka, if not set, `root` of the location is an empty dir (default nginx home)